### ISSUE TITLE
[Bug] Brick thrusters not firing

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#2197]
### Short Description of the issue:
When manoeuvring, brick thrusters placed for agility only fire visually when damping a ships movement, not when moving due to player input. This is just a cosmetic thing, but right now it's inconsistent.

### Steps to reproduce the issue:
-Pilot a ship with thrusters pointed up and down. (The same things happens for thrusters pointed left/right though too)
-Hold space to go up. Thrusters pointed downwards will not visually fire
-Release the space key, thrusters pointing up will fire to slow the ship down

### Environment
- Starship EVO version: 20w21c
- Graphics Card: RX 580

**Comment by tsunamayo** [comment]
Yes this is by design. Otherwise you would not see any effect for remote controlled ship (multi and AI). Plus the way it worked is actually newtonian => it fire on acceleration, not speed. If this is the problem I could always change that!
Thanks

**Comment by ProPeach** [comment]
Ah gotcha, I assumed they would fire when you strafe around then as they don't seem to fire when you first accelerate sideways in a strafe.
Personally I think they'd look better if they fired due to speed rather than acceleration, obviously that's not realistic but it doesn't look right or fit the more arcadey flight style as it stands
